src/widgets/courses/ui/courses.module.scss

```diff
@@ -0,0 +1,42 @@
+.all-courses {
+  @include media-laptop {
+    padding: 40px;
+  }
+
+  @include media-tablet {
+    padding: 0;
+  }
+}
+
+.courses-content {
+  padding-top: 40px;
+
+  @include media-laptop {
+    flex-wrap: wrap;
+    gap: 24px;
+    width: 100%;
+    padding: 8px 6px;
+  }
+
+  @include media-tablet {
+    padding: 0 16px;
+  }
+}
+
+.course-list {
+  display: grid;
+  grid-auto-flow: row;
+  grid-template: 1fr / repeat(3, 1fr);
+  gap: 22px;
+
+  width: 100%;
+  padding-top: 24px;
```

Comment:
this will fix the issue that @KristiBo described earlier

```suggestion

```

Reply:
Done